package.json

```diff
@@ -3,17 +3,17 @@
   "productName": "SolanaWorkbench",
   "description": "Solana workbench app for making development on Solana better",
   "scripts": {
+    "dev": "concurrently \"npm run start:main\" \"npm run start:renderer\"",
     "build": "concurrently \"npm run build:main\" \"npm run build:renderer\"",
-    "build:main": "cross-env NODE_ENV=production webpack --config ./.erb/configs/webpack.config.main.prod.ts",
-    "build:renderer": "cross-env NODE_ENV=production webpack --config ./.erb/configs/webpack.config.renderer.prod.ts",
-    "rebuild": "electron-rebuild --parallel --types prod,dev,optional --module-dir release/app",
+    "build:main": "tsc -p ./src/main/tsconfig.json",
+    "build:renderer": "vite build --config ./src/renderer/vite.config.ts",
+    "start:main": "npm run build:main && cross-env NODE_ENV=development electronmon -r ts-node/register/transpile-only ./src/main/main.ts",
+    "start:renderer": "vite dev --config ./src/renderer/vite.config.ts",
+    "package": "rimraf ./release && npm run build && electron-builder build --publish never",
+    "package:asarless": "npm run build && electron-builder build --config.asar=false",
     "lint": "cross-env NODE_ENV=development eslint . --ext .js,.jsx,.ts,.tsx",
-    "package": "ts-node ./.erb/scripts/clean.js dist && npm run build && electron-builder build --publish never",
-    "postinstall": "ts-node .erb/scripts/check-native-dep.js && electron-builder install-app-deps && cross-env NODE_ENV=development webpack --config ./.erb/configs/webpack.config.renderer.dev.dll.ts",
-    "start": "ts-node ./.erb/scripts/check-port-in-use.js && npm run start:renderer",
```

Comment:
its probably a nice thing to defined `npm run start` === `npm run dev` - for the new developer, the latter tells them little about what it does (does it start an editor?)